### PR TITLE
wa-standalone k8s

### PR DESCRIPTION
**Pull request by lewis150193**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: k8s/namespaces/wa/kustomization.yaml [hunk 1]
```diff
@@ -7,4 +7,5 @@ bases:
 - wa-task-management-api/wa-task-management-api.yaml
 - wa-task-configuration-api/wa-task-configuration-api.yaml
 - wa-proto-frontend/wa-proto-frontend.yaml
+- wa-standalone-task-bpmn/wa-standalone-task-bpmn.yaml
```
Comment (timja) [inline k8s/namespaces/wa/kustomization.yaml:10]: is this supposed to be in all environments? do you have the needed approvals?

File: k8s/namespaces/wa/kustomization.yaml [hunk 2]
```diff
@@ -7,4 +7,5 @@ bases:
 - wa-task-management-api/wa-task-management-api.yaml
 - wa-task-configuration-api/wa-task-configuration-api.yaml
 - wa-proto-frontend/wa-proto-frontend.yaml
```
Comment (timja) [inline k8s/namespaces/wa/kustomization.yaml:9]: is this supposed to be in prod?

Comment (lewis150193) [reply k8s/namespaces/wa/kustomization.yaml:9]: just preview

File: k8s/namespaces/wa/kustomization.yaml (hunk 1 above)
Comment (lewis150193) [reply k8s/namespaces/wa/kustomization.yaml:10]: its a new repo just used to hold tests

File: k8s/namespaces/wa/kustomization.yaml (hunk 2 above)
Comment (timja) [reply k8s/namespaces/wa/kustomization.yaml:9]: can you remove then please, and move to the appropriate place

File: k8s/namespaces/wa/kustomization.yaml (hunk 1 above)
Comment (timja) [reply k8s/namespaces/wa/kustomization.yaml:10]: then it shouldn't be going in to prod, see the line below

Comment (iguannago) [reply k8s/namespaces/wa/kustomization.yaml:10]: Hi @timja , yes, we need this service in Prod too. Thanks.

File: k8s/namespaces/wa/kustomization.yaml (hunk 2 above)
Comment (iguannago) [reply k8s/namespaces/wa/kustomization.yaml:9]: @timja , we have confirmed with Ravi, we this service needs on Prod too. Thanks

File: k8s/namespaces/wa/wa-standalone-task-bpmn/wa-standalone-task-bpmn.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  releaseName: wa-standalone-task-bpmn
+  chart:
+    git: git@github.com:hmcts/hmcts-charts
+    ref: master
+    path: stable/wa-standalone-task-bpmn
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/wa/workflow-api:latest
```
Comment (iguannago) [inline k8s/namespaces/wa/wa-standalone-task-bpmn/wa-standalone-task-bpmn.yaml:15]: @lewis150193 , this is not pointing to the right project

**Review by iguannago — COMMENTED** [review]
Please review.

**Review by iguannago — DISMISSED** [review]

**Review by luigibk — APPROVED** [review]

**Review by hannah38 — APPROVED** [review]

**Pull request merged**